monty/exts/info/github_info.py

```diff
@@ -908,6 +923,105 @@ async def swap_embed_state(self, inter: disnake.MessageInteraction) -> None:
 
         await inter.response.edit_message(embed=embed, components=rows)
 
+    async def handle_issue_comment(self, message: disnake.Message, issues: list[FoundIssue]) -> None:
+        """Expand an issue or pull request comment."""
+        comments = []
+        components = []
+
+        for issue in issues:
+            assert issue.url_fragment
+
+            # figure out which endpoint we want to use
+            frag = issue.url_fragment
+            if frag.startswith("issuecomment-"):
+                endpoint = ISSUE_COMMENT_ENDPOINT.format(
+                    user=issue.organisation,
+                    repository=issue.repository,
+                    comment_id=frag.removeprefix("issuecomment-"),
+                )
+            elif frag.startswith("pullrequestreview-"):
+                endpoint = PULL_REVIEW_COMMENT_ENDPOINT.format(
+                    user=issue.organisation,
+                    repository=issue.repository,
+                    comment_id=frag.removeprefix("pullrequestreview-"),
+                )
+            elif frag.startswith("discussion_r"):
+                endpoint = PULL_REVIEW_COMMENT_ENDPOINT.format(
+                    user=issue.organisation,
+                    repository=issue.repository,
+                    comment_id=frag.removeprefix("discussion_r"),
+                )
+            else:
+                continue
+
+            comment: dict[str, Any] = await self.fetch_data(endpoint, as_text=False)  # type: ignore
+            if "message" in comment:
+                log.warn("encountered error fetching %s: %s", endpoint, comment)
+                continue
+
+            # assert the url was not tampered with
+            if issue.user_url != (html_url := comment["html_url"]):
+                # this is a warning as its the best way I currently have to track how often the wrong url is used
+                log.warning("[comment autolink] issue url %s does not match comment url %s", issue.user_url, html_url)
+                continue
+
+            body = self.render_github_markdown(comment["body"])
+            e = disnake.Embed(
+                url=html_url,
+                description=body,
+            )
+
+            author = comment["user"]
+            e.set_author(
+                name=author["login"],
+                icon_url=author["avatar_url"],
+                url=author["html_url"],
+            )
+
+            e.set_footer(text=f"Comment on {issue.organisation}/{issue.repository}#{issue.number}")
+
+            e.timestamp = datetime.strptime(comment["created_at"], "%Y-%m-%dT%H:%M:%SZ")
+
+            comments.append(e)
+            components.append(disnake.ui.Button(url=comment["html_url"], label="View comment"))
+
+        if not comments:
+            return
+
+        if len(comments) > 4:
+            await message.reply(
```

Comment:
IMO not showing any of them makes more sense rather than the user possibly not noticing that one of the issues wasn't expanded. I'll revisit this later when i revisit integrations between all 5 of these link features. https://github.com/onerandomusername/monty-python/pull/228#discussion_r1155559534

Reply:
This is the same limitation (by design) which exists for github-style (user/repo#number) Issue embeds, so I'm keeping this limitation for now.